operator/tools/operator-sdk/go.mod

```diff
@@ -5,8 +5,8 @@ go 1.21
 toolchain go1.21.7
 
 require (
-	github.com/operator-framework/operator-lifecycle-manager v0.27.0
-	github.com/operator-framework/operator-sdk v1.35.0
+	github.com/operator-framework/operator-lifecycle-manager v3.11.0+incompatible
```

Comment:
This is wrong there is no such version:
https://github.com/operator-framework/operator-lifecycle-manager/tags